.sync.yml

```diff
@@ -18,6 +18,13 @@ spec/acceptance/nodesets/ec2/windows-2016-base-x64.yml:
 spec/acceptance/nodesets/archlinux-2-x64.yml:
   delete: true
 
+Gemfile:
+  optional:
+    ':development':
+      - gem: 'mocha'
```

Comment:
Did you not need this in the end?  I can't see the related change in the Gemfile